src/olympia/yara/models.py

```diff
@@ -26,6 +30,10 @@ def add_match(self, rule, tags=None, meta=None):
             'meta': meta or {},
         })
 
+    def save(self, *args, **kwargs):
+        self.has_matches = bool(self.matches)
```

Comment:
Is it ok to only compute this value here? For example, that's something I'd like to reuse in the task to send a statsd value. I guess I can do it after the `save()` but in the future, one might need this value before, maybe?

Reply:
I considered doing it in `add_match()` too, but I figured it wasn't enough, in case the `matches` were modified manually. I could do both though...

Reply:
0a1b4c9834d3d3d8024b742fe47e53aea23e6238